README.md

```diff
@@ -1,45 +1,37 @@
-[![Dependency Status](https://david-dm.org/raulg/reactjs-website.svg)](https://david-dm.org/raulg/reactjs-website)
+# Prismic & ReactJS Example Multi-Page Site
+> [ReactJS](https://reactjs.org/) example Multi-Page Site with content managed in [Prismic](https://prismic.io)
+## Check out the dedicated article to get this project up and running
+> [Prismic project guide](https://intercom.help/prismicio/en/articles/2731304-sample-multi-page-site-with-navigation-in-reactjs)
 
-# Prismic React Site Example
-This project is an implementation of the Prismic multipage site demo using ReactJS. Built up from the starter project, it's meant to provide you with an idea of how to build your own sites using the Prismic features. Read our [user guide](https://intercom.help/prismicio/en/articles/2731304-sample-multi-page-site-with-navigation-in-reactjs) if you need guidance on how to use this repository.
-
-## Getting started quickly with Prismic Themes
-You can use our [Prismic-CLI](https://github.com/prismicio/prismic-cli) tool to quickly install this project in your machine, as well as setting up a Prismic repository with the required Custom Types. Just follow the instructions in your terminal to get started!
+### 1. Install the prismic-cli
 
 ```
 npm install -g prismic-cli
-prismic theme https://github.com/raulg/reactjs-website --conf /src/prismic-configuration.js
 ```
 
-### Install dependencies
+### 2. Run the theme command
+This will create a new Prismic content repository, setup the custom types, and install the project code
```

Comment:
**I'd phrase it like:** 
This theme command will create a new repository in the Prismic dashboard(prismic.io/dashboard) with the custom-types and the contents. It also installs the project source code in your local.